main.go

```diff
@@ -11,13 +11,17 @@ import (
 
 func printHelp() {
 	fmt.Printf("Gosplat v1\n" +
-		" gosplat --h , prints commands\n" +
+		" gosplat -h/--help , prints this help text\n" +
```

Comment:
was fixed in another PR

Reply:
Don't think it was, i looked for it firstly in `dev`